bin/semantic-release.js

```diff
@@ -14,7 +14,7 @@ const { satisfies, lt } = semver;
 
 const MIN_GIT_VERSION = "2.7.1";
 
-if (!satisfies(process.version, engines.node)) {
+if (!satisfies(process.version, engines.node) && typeof Bun === "undefined") {
```

Comment:
I wonder if we should reverse the check, instead of checking if the runtime environment is Bun, we could check if the runtime environment is node